llm/generate/gen_common.sh

```diff
@@ -39,6 +39,9 @@ init_vars() {
     *)
         ;;
     esac
+    if [ -z "${CMAKE_CUDA_ARCHITECTURES}" ] ; then 
+        CMAKE_CUDA_ARCHITECTURES="50;52;61;70;75;80"
```

Comment:
Note: we may want to make this list "smart" about the cuda version detected, as there's a sliding window of EOL architectures.  (e.g., v12 drops 3.5 and 3.7 support)